ios/GigyaSdkWrapper.swift

```diff
@@ -312,6 +337,23 @@ class GigyaSdkWrapper<T: GigyaAccountProtocol>: GigyaSdkWrapperProtocol {
         }
     }
 
+    func sessionToDic(session: GigyaSession?) -> [String: Any]? {
```

Comment:
Conversion function is not needed - remove.

Reply:
Function is being called from getSession and setSession, so it would make sense to me to leave it here.

Reply:
I've modified it to use create the dictionary directly instead of relying on jsonEncoder